wradlib/util.py

```diff
@@ -1370,7 +1370,7 @@ def wrapper(decorated):
 
 def dim0(obj):
     """Return major dimension (azimuth/elevation) of xarray object."""
-    return list(set(obj.dims) & {"azimuth", "elevation"})[-1]
+    return list(set(obj.dims) & {"time", "azimuth", "elevation"})[-1]
```

Comment:
@egouden This checks for `azimuth`/`elevation` first and `time` second. This allows for things like:

```python
dim0 = {"time", "azimuth", "range"}
dim1 = {"time", "elevation", "range"}
dim2 = {"azimuth", "range"}
dim3 = {"elevation", "range"}
dim4 = {"time", "range"}
```

Reply:
This is a good quick fix !

However the underlying issue of the data model remains open. I think it should be clarified at the xradar level first.

Reply:
AFAICT, in xradar we have `azimuth`/`elevation` as default. The user has to explicitly request `time`. 

Beside that, many internal xarray functionality depends on having `azimuth`/`elevation` as first angle and raises accordingly. Note, this is a pure ergonomic representation.

Reply:
> AFAICT, in xradar we have `azimuth`/`elevation` as default. The user has to explicitly request `time`.
> 
> Beside that, many internal xarray functionality depends on having `azimuth`/`elevation` as first angle and raises accordingly. Note, this is a pure ergonomic representation.

Actually I got "time" as first dimension by using standard xradar/xarray tools. But strangely I cannot reproduce it after a refactor of my code. Therefore I thought that time was (going to be) the default in xradar in accordance to FM301. The fact that azimuth/elevation is currently the default in xradar is very interesting. Is there more information about this choice?

Reply:
It's more user friendly to have `azimuth`/`elevation` as first dimension instead of `time`. It's immediately visible what kind of sweep is is. When plotting you get `azimuth` vs `range` (or `elevation` vs `range`) instead of `time` vs `range`, which isn't that useful in the first place. Users might want to stack sweeps along another `time`-dimension (to get (`usertime`, `azimuth`, `range`)) which isn't possible out of the box with `time` as first dimension. That said, there are a multitude of reasons not to have `time` as first dimension.

Reply:
I think this choice of the dimension has a big impact on all user applications. Do you know if FM301 is already official? Should we not discuss this with WMO?

Reply:
From a applications point of view it doesn't matter IMHO. For our xarray based implementation this is just a switch of dimensions/coords. So I'd not worry too much about that. FM301 is official (see [Weather Radar Data Representation and Exchange](https://library.wmo.int/records/item/68829-guide-to-operational-weather-radar-best-practices) and [Manual of Codes](https://library.wmo.int/records/item/35625-manual-on-codes-volume-i-2-international-codes)).